service/resource/legacyv2/cluster_role_binding.go

```diff
@@ -39,8 +39,8 @@ func newClusterRoleBinding(customObject v1alpha1.FlannelConfig) *v1beta1.Cluster
 func newClusterRoleBindingForDeletion(customObject v1alpha1.FlannelConfig) *v1beta1.ClusterRoleBinding {
 	config := ClusterRoleBindingConfigDefaultConfig()
 	config.name = clusterRoleBindingForDeletion(customObject.Spec)
-	config.subjectName = clusterID(customObject.Spec)
-	config.subjectNamespace = networkNamespace(customObject.Spec)
+	config.subjectName = serviceAccountName(customObject.Spec)
+	config.subjectNamespace = destroyerNamespace(customObject.Spec)
 	config.roleName = "flannel-operator"
```

Comment:
Unifying names as they live in different namespaces and it has more sense a common naming